File: src/logic/NPC.java
package logic;

public class NPC {
    public enum Type {
        Friendly,
        Hostile
    }

    Type type;
    String name;
    int hp;

    public NPC(Type type, String name, int hp) {
        this.type = type;
        this.name = name;
        this.hp = hp;
    }

    public enum NPCs {
        LoreMaster(new NPC(Type.Friendly, "Ghernard of Logaria", 100));

        public NPC npc;

        NPCs(NPC npc) {
            this.npc = npc;
        }

    }

}

File: Game/NPC.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Game
{
    public enum NPCs
    {
        LoreMaster
    }
    public class NPC
    {
        public enum Type
        {
            Friendly,
            Hostile
        }

        Quest[] quests;
        Type type;
        string name;
        int hp;

        public NPC(Type type, string name, int hp)
        {
            this.type = type;
            this.name = name;
            this.hp = hp;
        }

        //quests will just be bools and strings for hints that get set in some places

    }
}